src/test/scholarshipsTest.js

```diff
@@ -0,0 +1,54 @@
+const scholarships = [
```

Comment:
Regarding tests, I think it'd be fine to leave tests next to the source file they're testing (let me know if you think differently). In either case, this file is not really a test, but rather it has test data so maybe `src/testdata/scholarships.js`?